sys/androidmedia/gstamcvideodec.c

```diff
@@ -490,6 +451,20 @@ create_src_caps (const GstAmcCodecInfo * codec_info, gboolean direct_rendering)
   return ret;
 }
 
+static GstFlowReturn
+gst_amc_video_dec_push_dummy (GstAmcVideoDec * self)
+{
+  GstBuffer *buf = gst_buffer_new ();
+  GstCaps *caps = gst_caps_new_simple ("video/x-amc", NULL);
```

Comment:
This is a small, but still a performance fall: we're creating empty "video/x-amc" caps each time (it adds several malloc/free), where we could just create it once.